pmg/templates/_top_links.html

```diff
@@ -1,5 +1,46 @@
-<li><a href="/committees/">Committees</a></li>
-<li><a href="/members">MPs</a></li>
-<li><a href="/bills">Bills</a></li>
-<li><a href="/question_replies">Questions and Replies</a></li>
-<li><a href="/calls-for-comments">Calls for Comments</a></li>
+<div class="menu-container hidden-xs">
```

Comment:
this should be inside an li - onlu li's can be children of ul's https://stackoverflow.com/questions/11755628/can-i-use-div-as-a-direct-child-of-ul